utils/src/workspaceConfig.ts

```diff
@@ -0,0 +1,3 @@
+import { workspace, WorkspaceConfiguration } from "vscode";
+
+export const workspaceConfig: WorkspaceConfiguration = workspace.getConfiguration();
```

Comment:
Yeah, it works when I run it at least.  It might work both ways.  I'd be cool with changing it to that though, it seems like it would make more sense to scope it like that rather than load the entire settings file, what do you think?

Reply:
Good idea. I'm not entirely sure how VSCode handles the settings at runtime but since we get a specific section everywhere else in our code, it's probably best to do so here too.